src/routes/current-studies/StudyCard.svelte

```diff
@@ -137,18 +140,12 @@ $: isActive = $activeKey !== undefined && $activeKey === key;
     </div>
     <div class:split-content-modal={joined} slot="body">
         {#if !joined}
-            {#if joinStudyConsentNotice}
-                {@html joinStudyConsentNotice}
-            {:else}
-            <!-- default "join study" consent notice -->
-            <p>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-                eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
-                enim ad minim veniam, ut enim ad minim veniam, ut enim ad minim
-                veniam.
-                </p>
-                <p>TBD approach based on explorations with SimplySecure.</p>
-            {/if}
+            <!-- Bake in the Princeton IRB. Once we have more studies, we will key this
+                 by the study id.
+            -->
+            <IRBWindow>
+                <svelte:component this={irb['princeton-study']} />
+            </IRBWindow>
```

Comment:
Can we do this just for the princeton addon id? This way we could still test other consent notices in our local builds.

Reply:
Yes, that is the ultimate intent. Given that we don't have any other studies or consent notices, and given that we don't have the princeton addon id quite yet to do this part, I'm inclined to leave this as-is until we have more studies and more bandwidth to redesign how we're doing this part.

Do you have any suggestions on how to proceed?

Reply:
Ok